source/video_chunk_lambda/init.py

```diff
@@ -0,0 +1,105 @@
+
+import os
+import boto3
+import json
+import logging
+import botocore
+import cv2
+from helpers import get_secrets
+
+secret_name = os.getenv("SECRETS_NAME")
+secrets = get_secrets(secret_name)
+
+def lambda_handler(event, context):
+    s3 = boto3.client('s3')
+    print(event)
+    
+    try:
+        logger = logging.getLogger()
+        logger.setLevel(logging.INFO)
+        logger.info('event parameter: {}'.format(event))
+        tmp_file_path = "/tmp/video.mp4"
```

Comment:
Could this generate a collision if two lambdas were executed at the same time?
```suggestion
        import uuid

        tmp_file_path = "/tmp/video_" + uuid.uuid4().hex + ".mp4"
```